src/chrome/content/rules/NYTimes-Problematic.xml

```diff
@@ -0,0 +1,114 @@
+<!--
+	For other New York Times coverage, see NYTimes.xml.
+-->
+<ruleset name="NYTimes-Problematic">
+	<!-- As of 4/2017, 5th level .blogs.nytimes.com domains do not support SSL.
+		https://this.host.blogs.nytimes.com/
+		https://exists.not.blogs.nytimes.com/
+			-->
+	<target host="*.blogs.nytimes.com" />
+	<rule from="^http://([\w-]+)\.blogs\.nytimes\.com/" to="https://$1.blogs.nytimes.com/" />
+		<test url="http://firstlook.blogs.nytimes.com/" />
+		<test url="http://lens.blogs.nytimes.com/" />
+		<test url="http://newoldage.blogs.nytimes.com/" />
+
+	<!--	Redirect keeps path, but not args nor forward slash:	-->
+	<target host="documents.nytimes.com" />
+	<rule from="^http://documents\.nytimes\.com/+([^?]*)"
+		to="https://www.nytimes.com/interactive/projects/documents/$1" />
+		<test url="http://documents.nytimes.com/index.htm" />
+		<test url="http://documents.nytimes.com/index.html" />
+
+	<!--	Redirect drops path and forward slash, but not args:	-->
+	<target host="jobmarket.nytimes.com" />
+	<rule from="^http://jobmarket\.nytimes\.com/[^?]*"
+		to="https://www.nytimes.com/section/jobs" />
+		<test url="http://jobmarket.nytimes.com/index.html" />
+
+	<target host="pimage.timespeople.nytimes.com" />
+	<rule from="^http://pimage\.timespeople\.nytimes\.com/"
+		to="https://s3.amazonaws.com/pimage.timespeople.nytimes.com/" />
+
+	<!--	Too many redirects:	-->
+	<target host="regilite.nytimes.com" />
+	<rule from="^http://regilite\.nytimes\.com/regilite\?product="
+		to="https://www.nytimes.com/newsletters/regilite/button?product=" />
+		<test url="http://regilite.nytimes.com/regilite?product=TY" />
+
+	<rule from="^http://regilite\.nytimes\.com/$" to="https://www.nytimes.com/" />
+
+	<!--	Redirect drops args and forward slash:	-->
+	<target host="travel.nytimes.com" />
+	<rule from="^http://travel\.nytimes\.com/(?:$|\?.*)"
+		to="https://www.nytimes.com/section/travel" />
+		<test url="http://travel.nytimes.com/?" />
+		<test url="http://travel.nytimes.com//?" />
+
+	<rule from="^http://travel\.nytimes\.com/"
+		to="https://www.nytimes.com/" />
+		<test url="http://travel.nytimes.com/section/realestate" />
```

Comment:
Could you also add a test url with the `travel.nytimes.com/YYYY/MM/DD/opinion/\w+.html` scheme?